electricitylci/modelconfig/ELCI_1_config.yml

```diff
@@ -12,7 +12,7 @@ electricity_lci_target_year: 2018
 # Select a regional aggregation from "eGRID", "NERC", "BA", "US", "FERC", and "EIA"
 # The EPA_eGRID trading method can only be used with "eGRID".
 # The NETL_BA_trading method can only be used with "BA" and "FERC"
-regional_aggregation: 'FERC'
+regional_aggregation: 'BA'
```

Comment:
@bl-young @jump2conclusionsmatt I had noted this region setting to FERC in ELCI_1 before and was worried about changing it. 
For instance, what will the effect be in these places
[eia860_facilities.py#L156](https://github.com/USEPA/ElectricityLCI/blob/ab858ca49a0514000ce845f9748b8f6470d3f56e/electricitylci/eia860_facilities.py#L156)

[generation_mix.py#L121](https://github.com/USEPA/ElectricityLCI/blob/ab858ca49a0514000ce845f9748b8f6470d3f56e/electricitylci/generation_mix.py#L121)

[main.py#L42](
https://github.com/USEPA/ElectricityLCI/blob/ab858ca49a0514000ce845f9748b8f6470d3f56e/electricitylci/main.py#L42)

[main.py#L42](https://github.com/USEPA/ElectricityLCI/blob/ab858ca49a0514000ce845f9748b8f6470d3f56e/electricitylci/main.py#L57)

Reply:
That setting won't really have any consequences on what will get generated for the json-ld. If that setting is FERC, BA, or US, generation mixes for balancing authority areas will created because that is how the trading model is setup - to always trade with BAAs. And then, the model always creates consumption mixes for BA, FERC, and US because it was such a small computational lift to do so.

Reply:
I figured it would be best to match what was listed on the release notes. So sounds like Ok to leave as is?

Reply:
I'm okay with BA.

Reply:
BA is the accurate value. I was just worried of the consequences in the code. But @jump2conclusionsmatt confirmed also that it will have the same result, so let's leave it.